src/comparisons.jl

```diff
@@ -87,9 +87,7 @@ A boolean that is `true` iff `x ≈ 0`.
 This is a fallback implementation for any real `x` such that `x ≈ 0` is `true`
 whenever `x` is equal to zero in the same numeric type as `x`.
 """
-function isapproxzero(x::N; kwargs...) where {N<:Real}
-    return x == zero(N)
-end
+isapproxzero(x::Real; kwargs...) = x == zero(N)
```

Comment:
`N` is not defined here.

Reply:
Ouch, fixed. I should add a comparisons unit tests file.

Reply:
definitely :wink: